### ISSUE TITLE
Multiple messages when joining a tribe

### ISSUE THREAD

**Issue by ckang03** [stakwork/sphinx-tribes#245]
When joining a new tribe (this one specifically required an admin to admit me), all of messages prior to me joining are showing up 4 times. The "welcome! You're now a member" also shows up 4 times, and the "___ has joined the tribe" shows up 13 times. 
<img width="1079" alt="Screen Shot 2022-07-18 at 12 20 02 PM" src="https://user-images.githubusercontent.com/108751187/179580176-0864832a-fb77-48a5-a8bf-a47a9ea1834a.png">
<img width="1079" alt="Screen Shot 2022-07-18 at 12 20 26 PM" src="https://user-images.githubusercontent.com/108751187/179580180-ce60f318-0b69-4528-ac01-deea8ff7bafc.png">
<img width="1087" alt="Screen Shot 2022-07-18 at 12 20 11 PM" src="https://user-images.githubusercontent.com/108751187/179580183-40a65a28-2c89-4dc7-9bda-b07d639c516c.png">

**Comment by kevkevinpal** [comment]
@ckang03 
do you see this happening when you join a tribe when the admin doesn't need to admit you?

**Comment by ckang03** [comment]
@kevkevinpal 
No. This was the first tribe I've joined where the admin needs to admit me if that helps.
